aidb/engine/base_engine.py

```diff
@@ -462,14 +462,6 @@ def execute_user_defined_function(
     df_query = f'''SELECT {select_str} FROM processed_udf_result_df {where_str}'''
 
     new_results_df = duckdb.sql(df_query).df()
-    res_list_of_tuple = []
-    for row in new_results_df.itertuples(index=False):
-      row_list = []
-      for item in row:
-        if isinstance(item, tuple) or isinstance(item, list):
-          row_list.extend(list(item))
-        else:
-          row_list.append(item)
-      res_list_of_tuple.append(tuple(row_list))
+    res_list_of_tuple = [tuple(row) for row in new_results_df.itertuples(index=False)]
```

Comment:
For UDF that has multiple columns output, we have expanded it to multiple columns in self._get_udf_result(). So we don't need to process this again.